pam/internal/adapter/nativemodel.go

```diff
@@ -584,13 +584,13 @@ func (m nativeModel) handleFormChallenge(hasWait bool) tea.Cmd {
 		}
 	}
 
-	instructions := "Insert '%[1]s' to cancel the request and go back"
+	instructions := "Type '%[1]s' to cancel the request and go back"
 	if hasWait {
 		// Duplicating some contents here, as it will be better for translators once we've them
-		instructions = "Leave the input field empty to wait for other authentication method " +
+		instructions = "Leave the input field empty to wait for the alternative authentication method " +
 			"or insert '%[1]s' to go back"
 		if m.uiLayout.GetEntry() == "" {
-			instructions = "Leave the input field empty to wait for the authentication method " +
+			instructions = "Press Enter to wait for the authentication method " +
 				"or insert '%[1]s' to go back"
```

Comment:
```suggestion
				"or type '%[1]s' to go back"
```

Reply:
I'm still undecided if we should still use `enter` in all these cases instead, since `type` may imply that there's no expected confirmation from the user, while indeed one has to type `r` and press enter too. :thinking:

Reply:
Ha, I just wanted to suggest the same. I tried out the UX and also thought that "Enter 'r' to cancel the request and go back" would make it more clear to the user that they need to type "r" instead of the password / number of the authentication method.